handbook/ops/bizops/amplitude.md

```diff
@@ -0,0 +1,29 @@
+## How do I get started?
+1. [Understand the data](#data)
+1. Amplitude documentation for [building analyses](https://help.amplitude.com/hc/en-us/categories/360003165371-Build-and-share-your-analysis)
+1. Ask BizOps!
+
+## Why Amplitude?
+Amplitude is a specialized product analytics tool that specializes in turning event data into actionable insights and dashboards. 
+
+### Why aren’t we using Looker for this?
+Looker is very flexible in that we have the ability to set it up for any purpose. But this also means that to provide a great self-service experience for what we want from a product analytics tool, this would take a LOT of work. We get these capabilities out of the box with Amplitude. 
+
+### What is in Looker vs. Amplitude?
+
+Anything not based directly on analyzing events is in Looker. This includes pings from on-prem instances, anything we get from the Cloud Postgres database, and any data from third-parties tools (such as Google Analytics and Salesforce). 
+
+| Type of analysis        | Tool      | Example |
+|-------------------------|-----------|---------|
+| Retention/engagement    | Amplitude | [Link](https://analytics.amplitude.com/sourcegraph/chart/7l5vdg4?source=workspace)    |
+| Adoption of public code | Looker    | [Link](https://sourcegraph.looker.com/dashboards-next/175)    |
+| Progress towards OKRs   | Looker    | [Link](https://sourcegraph.looker.com/dashboards-next/166)    |
+| On-prem instances/pings | Looker    | [Link](https://sourcegraph.looker.com/dashboards-next/174)    |
+| Signup funnel on Cloud  | Amplitude | [Link](https://analytics.amplitude.com/sourcegraph/chart/jumvevm?source=workspace)    |
+
+
```

Comment:
Is this accurate? If so, I think we should state it. I think the first question people have when something new comes in is "how will this change my current workflow?" 
```suggestion
Any analysis that was conducted in Looker prior to the implementation of Amplitude can still be done in Looker, and existing Looker dashboards and visualizations will still be maintained. Amplitude will help us conduct new and different analysis regarding product analytics. 
```